code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -156,14 +164,13 @@
 //Updates an organ's brute/burn states for use by update_damage_overlays()
 //Returns 1 if we need to update overlays. 0 otherwise.
 /obj/item/bodypart/proc/update_bodypart_damage_state()
-	if(status == ORGAN_ORGANIC) //Robotic limbs show no damage - RR
-		var/tbrute	= round( (brute_dam/max_damage)*3, 1 )
-		var/tburn	= round( (burn_dam/max_damage)*3, 1 )
-		if((tbrute != brutestate) || (tburn != burnstate))
-			brutestate = tbrute
-			burnstate = tburn
-			return 1
-		return 0
+	var/tbrute	= round( (brute_dam/max_damage)*3, 1 )
+	var/tburn	= round( (burn_dam/max_damage)*3, 1 )
+	if((tbrute != brutestate) || (tburn != burnstate))
+		brutestate = tbrute
+		burnstate = tburn
+		return 1
+	return 0
```

Comment:
robotic limbs get a brutestate and burnstate since they now get a damage overlay.